label_studio/tests/utils.py

```diff
@@ -124,8 +124,8 @@ def generate_signed_url(self, **kwargs):
         def download_as_bytes(self):
```

Comment:
slightly DRY-er approach

```
def download_as_bytes(self):
    data = json.dumps(self.sample_json_contents) if self.is_json else f'test_blob_{self.key}'
    return data.encode()
```

Reply:
would agree in isolation, but want to make the parallel with `download_as_string` very obvious